cluster/manifests/dashboard/rbac.yaml

```diff
@@ -3,11 +3,92 @@ kind: ServiceAccount
 metadata:
   name: kubernetes-dashboard
   namespace: kube-system
+  labels:
+    application: kubernetes-dashboard
+
 ---
-apiVersion: rbac.authorization.k8s.io/v1beta1
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: kubernetes-dashboard
+  namespace: kube-system
+  labels:
+    application: kubernetes-dashboard
+rules:
+# Allow Dashboard to get, update and delete Dashboard exclusive secrets.
+- apiGroups: [""]
+  resources: ["secrets"]
+  resourceNames: ["kubernetes-dashboard-key-holder", "kubernetes-dashboard-certs", "kubernetes-dashboard-csrf"]
+  verbs: ["get", "update", "delete"]
+  # Allow Dashboard to get and update 'kubernetes-dashboard-settings' config map.
+- apiGroups: [""]
+  resources: ["configmaps"]
+  resourceNames: ["kubernetes-dashboard-settings"]
+  verbs: ["get", "update"]
+  # Allow Dashboard to get metrics.
+- apiGroups: [""]
+  resources: ["services"]
+  resourceNames: ["dashboard-metrics-scraper"]
+  verbs: ["proxy"]
+- apiGroups: [""]
+  resources: ["services/proxy"]
+  resourceNames: ["dashboard-metrics-scraper", "http:dashboard-metrics-scraper"]
+  verbs: ["get"]
+
+---
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: kubernetes-dashboard
+  labels:
+    application: kubernetes-dashboard
+rules:
+# Allow Metrics Scraper to get metrics from the Metrics server
```

Comment:
That looks like an extremely limited role. How will the users be able to interact with deployments, services, etc?